### PR TITLE
Login Redux

### PR DESCRIPTION
**Pull request by sshaul**
Added redux to login page

### CODE REVIEW

File: Mobile/stockdogmobile/reducers/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { ActionConst } from 'react-native-router-flux';
+import { combineReducers } from 'redux';
+
+// import the other reducers
+import loginReducer from './loginReducer';
+
+const sceneReducer = (state = {}, {type, scene}) => {
+    console.log('in scene reducer');
+    console.log(type);
```
Comment (asnewman) [inline Mobile/stockdogmobile/reducers/index.js]: Do you want to keep these log statements?

File: Mobile/stockdogmobile/reducers/loginReducer.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { ACTION_TYPES } from '../actions/actionTypes';
+
+export default loginReducer = (state = {user: null, password: null}, action) => {
+    console.log('in login reducer: ', action.type);
+    switch(action.type) {
+        case ACTION_TYPES.LOGIN_USER:
+            console.log('logging in: ', action);
```
Comment (asnewman) [inline Mobile/stockdogmobile/reducers/loginReducer.js]: More logs

File: Mobile/stockdogmobile/routes.js [hunk 3]
```diff
@@ -19,54 +22,60 @@ import containers from './style/containers';
...
+                </Lightbox>
+              
+          
+          <Lightbox>
+            <Scene key="stock" component={Stock} hideNavBar swipeDownToClose={false}/>
+            <Scene key="buysellmodal" component={BuySellModal} hideNavBar />
+          </Lightbox>
+          <Scene key="settings" component={SettingsModal} hideNavBar />
+        </Modal>
+      </Drawer> */}
```
Comment (asnewman) [inline Mobile/stockdogmobile/routes.js]: Why is the drawer commented out?

File: Mobile/stockdogmobile/style/elements.js [hunk 4]
```diff
@@ -8,7 +8,7 @@ export default elements = StyleSheet.create({
         backgroundColor: 'transparent',
         // borderRadius: 12,
         borderBottomWidth: 1,
-        borderColor: colors.placeholders,
+        borderColor: '#929292',
```
Comment (asnewman) [inline Mobile/stockdogmobile/style/elements.js]: Is this a color that's used globally?

File: Mobile/stockdogmobile/routes.js (hunk 3 above)
Comment (sshaul) [reply Mobile/stockdogmobile/routes.js]: I wanted to start off with only the pages that are being converted to the new styles and use redux but I still wanted to keep the router that I had so I can incrementally add the pages back in without having to redo the whole router.

File: Mobile/stockdogmobile/style/elements.js (hunk 4 above)
Comment (sshaul) [reply Mobile/stockdogmobile/style/elements.js]: Oops thanks for catching that!

**Pull request merged**